### PR TITLE
[Topic Entry] Python math-module math.cosh()

### PR DESCRIPTION
**Pull request by FiFiTM**
### Description

- Added the math.cosh() topic to Python math-module

### Issue Solved

- Closes Issue #5083

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-cosh/math-cosh.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Syntax
+
+```pseudo
+math.cosh(x)
+```
+
+The `math.cosh()` method is equivalent to _(exp(number) + exp(-number)) / 2_.
+
+## Example
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-cosh/math-cosh.md:25]:
```suggestion
## Example 1
```

File: content/python/concepts/math-module/terms/math-cosh/math-cosh.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+1.5430806348152437
+```
+
+Use `math.cosh()` to return the hyperbolic cosine of the float `8.9`:
+
+```py
+import math
+
+print(math.cosh(8.9))
+```
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-cosh/math-cosh.md:47]: After a heading as Example 2

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi, @FiFiTM thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @FiFiTM 😄 

The entry is ready to be merged 🚀

**Pull request merged**